subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -453,6 +454,23 @@ public void setDebug(boolean enabled) {
         forkOptions.setDebug(enabled);
```

Comment:
❓ We should deprecate and schedule the `getDebug`/`setDebug` method to be removed.

Reply:
I'm going to create an issue/PR for that.